test/functional/test-url-replacements.js

```diff
@@ -249,7 +250,7 @@ describes.sandboxed('UrlReplacements', {}, () => {
         resolve();
       });
     });
-    return installUrlReplacementsServiceForDoc(win.ampdoc)
+    return getServiceForDoc(win.ampdoc, 'url-replace')
```

Comment:
Why not add a new `urlReplaceForDoc` function to `services.js`? It'll avoid the risk of typos in the service name.